### ISSUE TITLE
update vector-log-proxy pulumi code to use the OLAutoScaling component resource(s)

### ISSUE THREAD

**Issue by Ardiea** [mitodl/ol-infrastructure#1424]
# Description/Context
This stack is a particularly egregious example of complicated and long autoscaling config located within the stack code itself. We can chop this in half or more by migrating it to the OLAutoScaling component resource. Blocked by https://github.com/mitodl/ol-infrastructure/issues/1423

# Acceptance Criteria
- [ ]

**Comment by blarghmatey** [comment]
Closing this in favor of migrating this use case to run on Kubernetes
